sample_size/scripts/input_utils.py

```diff
@@ -1,11 +1,9 @@
+from typing import Any
 from typing import Dict
-from typing import Tuple
+from typing import List
 
 from sample_size.sample_size_calculator import DEFAULT_ALPHA
 
-# Consider simplify the code using https://github.com/pallets/click/
```

Comment:
Is there a reason we're not using click?  Seems like a great use-case

Reply:
yeah, let's at least have a TODO if we are planning on doing it later